ENV.MD

```diff
@@ -0,0 +1,427 @@
+
+##### SPOOF_PROTECTION
+Configures the handling of creating mails with forged sender addresses.
+  - **empty** => Mail address spoofing allowed. Any logged in user may create email messages with a forged sender address. See also [Wikipedia](https://en.wikipedia.org/wiki/Email_spoofing)(not recommended, but default for backwards compatibility reasons)
+  - 1 => (recommended) Mail spoofing denied. Each user may only send with his own or his alias addresses. Addresses with [extension delimiters](http://www.postfix.org/postconf.5.html#recipient_delimiter) are not able to send messages.
+
+##### ENABLE_SRS
+Enables the Sender Rewriting Scheme. SRS is needed if your mail server acts as forwarder. See [postsrsd](https://github.com/roehling/postsrsd/blob/master/README.md#sender-rewriting-scheme-crash-course) for further explanation.
+  - **0** => Disabled
+  - 1 => Enabled
+
+##### PERMIT_DOCKER
+
+Set different options for mynetworks option (can be overwrite in postfix-main.cf)
+  - **empty** => localhost only
+  - host => Add docker host (ipv4 only)
+  - network => Add the docker default bridge network (172.16.0.0/12); **WARNING**: `docker-compose` might use others (e.g. 192.168.0.0/16) use `PERMIT_DOCKER=connected-networks` in this case
+  - connected-networks => Add all connected docker networks (ipv4 only)
+
+##### VIRUSMAILS_DELETE_DELAY
+
+Set how many days a virusmail will stay on the server before being deleted
+  - **empty** => 7 days
+
+
+##### ENABLE_POSTFIX_VIRTUAL_TRANSPORT
+
+This Option is activating the Usage of POSTFIX_DAGENT to specify a ltmp client different from default dovecot socket.
+
+- **empty** => disabled
+- 1 => enabled
+
+##### POSTFIX_DAGENT
+
+Enabled by ENABLE_POSTFIX_VIRTUAL_TRANSPORT. Specify the final delivery of postfix
+
+- **empty**: fail
+- `lmtp:unix:private/dovecot-lmtp` (use socket)
+- `lmtps:inet:<host>:<port>` (secure lmtp with starttls, take a look at https://sys4.de/en/blog/2014/11/17/sicheres-lmtp-mit-starttls-in-dovecot/)
+- `lmtp:<kopano-host>:2003` (use kopano as mailstore)
+- etc.
+
+##### POSTFIX\_MAILBOX\_SIZE\_LIMIT
+
+Set the mailbox size limit for all users. If set to zero, the size will be unlimited (default).
+
+- **empty** => 0 (no limit)
+
+##### POSTFIX\_MESSAGE\_SIZE\_LIMIT
+
+Set the message size limit for all users. If set to zero, the size will be unlimited (not recommended!)
+
+- **empty** => 10240000 (~10 MB)
+
+##### ENABLE_MANAGESIEVE
+
+  - **empty** => Managesieve service disabled
+  - 1 => Enables Managesieve on port 4190
+
+##### OVERRIDE_HOSTNAME
+
+  - **empty** => uses the `hostname` command to get the mail server's canonical hostname
+  - => Specify a fully-qualified domainname to serve mail for.  This is used for many of the config features so if you can't set your hostname (e.g. you're in a container platform that doesn't let you) specify it in this environment variable.
+
+##### POSTMASTER_ADDRESS
+
+  - **empty** => postmaster@domain.com
```

Comment:
is that then really postmaster@domain.com or is domain.com depending on the hostname?